database/migration/changesets/20200316_ExternalWholeGenomeReprocessing.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="tbl@broadinstitute.org"
+               dbms="postgresql"
+               id="20200316"
+               logicalFilePath="20200316_ExternalWholeGenomeReprocessing.xml">
+        <comment>
+            Define the ExternalWholeGenomeReprocessing table schema.
+            The name of an instance of this type of table
+            appears in the 'workload.table' column.
+            There is a row in this table for every workflow
+            that is part of the workload named by this table.
+        </comment>
+        <rollback>
+            I have no idea.
+        </rollback>
+        <sql>
+            ALTER TYPE workflow ADD VALUE 'ExternalWholeGenomeReprocessing'
+            <comment>
+                The 'workload.pipeline' column value
+                for tables of this kind.
+            </comment>
+        </sql>
+        <sql>
+            CREATE TYPE ExternalWholeGenomeReprocessing AS (
+            id                     bigint, -- unique to this table
+            base_file_name         text,   -- input to the workflow
+            cromwell               text,   -- the Cromwell metadata URL
+            destination_cloud_path text,   -- input to the workflow
+            final_gvcf_base_name   text,   -- input to the workflow
+            input_cram             text,   -- input to the workflow
+            sample_name            text,   -- input to the workflow
+            unmapped_bam_suffix    text)   -- input to the workflow
```

Comment:
❤️ Nice, I think for now these fields are sufficient to provide flexible input options.